lib/src/rules/unnecessary_overrides.dart

```diff
@@ -227,13 +237,23 @@ class _UnnecessaryMethodOverrideVisitor
   _UnnecessaryMethodOverrideVisitor(super.rule);
 
   @override
-  ExecutableElement? getInheritedElement(node) => node.lookUpInheritedMethod();
```

Comment:
Do we still use `lookUpInheritedMethod()`?

Reply:
We have in `InterfaceType`:
```dart
  MethodElement? lookUpMethod2(
    String name,
    LibraryElement library, {
    bool concrete = false,
    bool inherited = false,
    bool recoveryStatic = false,
  }) {
```

So, I wonder if `thisType.lookUpMethod2(name, library, concrete: true, inherited: true)` would work.

Reply:
Thanks that works perfectly!